Content.Server/Explosion/EntitySystems/TriggerSystem.cs

```diff
@@ -189,6 +191,12 @@ private void HandleRattleTrigger(EntityUid uid, RattleComponent component, Trigg
 
         private void OnTriggerCollide(EntityUid uid, TriggerOnCollideComponent component, ref StartCollideEvent args)
         {
+            //Don't trigger if we collide with an entity that's laying down and is not our target.
+            if (TryComp<TargetedProjectileComponent>(uid, out var targetedProjectile) &&
+                args.OtherFixture.CollisionLayer == (int) CollisionGroup.LayingDownMobLayer &&
+                targetedProjectile.Target != args.OtherEntity)
+                return;
+
```

Comment:
rather than special casing it for every projectile collision imo best way to handle this is to just have `ProjectileSystem` then raise its own event like `ProjectileCollidedEvent` that can be subbed to

Reply:
I made an event, but I have to raise it in both TriggerSystem and ProjectileSystem.